src/axes.jl

```diff
@@ -309,15 +309,13 @@ function expand_extrema!(sp::Subplot, d::KW)
 
     # expand for bar_width
     if d[:seriestype] == :bar
-        dsym = vert ? :x : :y
+        dsym = (!vert) ? :x : :y
```

Comment:
Well, it was skewed in the wrong direction so I changed this and then it was working...